man/org.freedesktop.systemd1.xml

```diff
@@ -1286,6 +1292,13 @@ node /org/freedesktop/systemd1 {
       <varname>ExecStop=</varname> and is spawned in parallel to the main daemon process in order to shut it
       down.</para>
 
+      <para><function>QueueSignalUnit()</function> is similar to <function>KillUnit()</function> but may be
+      used to enqueue a POSIX Realtime Signal (i.e. <constant>SIGRTMIN+…</constant> and
+      <constant>SIGRTMAX-…</constant>) to the selected process(es). Takes the same paramaters as
+      <function>KillUnit()</function> with one additional argument: an integer that is passed in the
+      <varname>sival_int</varname> value accompanying the queued signal. See <citerefentry project="man-pages"><refentrytitle>sigqueue</refentrytitle><manvolnum>3</manvolnum></citerefentry> for
```

Comment:
So, this is actually "ninja -C build update-dbus-docs" which rewrites it like that, and the D-Bus docs validity checks will refuse otherwise. I'll leave it like this for now, to make that stuff happy, but this is definitely something to fix in the the "update-dbus-docs" logic.

/cc @keszybz